src/config/configurator.ts

```diff
@@ -112,12 +112,14 @@ interface getStackSynthesizerProps {
 export function getStackSynthesizer(props: getStackSynthesizerProps): cdk.IStackSynthesizer {
   const configData = getConfig({ config: props.config });
 
-  const accountId =
-    configData && props.environmentId ? configData.environments[props.environmentId].account : defaultAccountId;
-
-  const region =
-    configData && props.environmentId ? configData.environments[props.environmentId].region : defaultRegion;
-
+  try {
+    var accountId =
```

Comment:
Did you need to change this `const` to `var` because you were putting it in the try-catch block?

This is a good opportunity to use `let`. Unlike `var`, it's still limits the scope to the block. However, unlike `const`, you can declare it without initialization outside the block.

```typescript
let accountId;
let region;

try {
  accountId = ...
  region = ...
}
```

Reply:
Yes, and appreciate the suggestion. Will try that 😄